checks/apps/cp2k/cp2k_uenv.py

```diff
@@ -26,7 +34,14 @@
             'cpus-per-task': 16,
             'walltime': '0d0h5m0s',
             'gpu': True,
-        }
+        },
+        'zen2': {
+            'nodes': 1,
+            'ntasks-per-node': 64,
+            'cpus-per-task': 2,
```

Comment:
64 ranks and 2 threads is realistic. However, because of the OpenBLAS extra thread, it actually means we run the test single threaded. For coverage, maybe it's better to use 32/4 ?

Reply:
Good point, it might indeed be better to run OpenBLAS with multiple threads.